Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Types/DriverFee.hs

```diff
@@ -0,0 +1,41 @@
+{-
+ Copyright 2022-23, Juspay India Pvt Ltd
+
+ This program is free software: you can redistribute it and/or modify it under the terms of the GNU Affero General Public License
+
+ as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version. This program
+
+ is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+
+ or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License for more details. You should have received a copy of
+
+ the GNU Affero General Public License along with this program. If not, see <https://www.gnu.org/licenses/>.
+-}
+
+module Domain.Types.DriverFee where
+
+import Data.Time (UTCTime)
+import Domain.Types.Person (Driver)
+import EulerHS.Prelude (Eq, Generic, Int, Read, Show)
+import Kernel.Types.Common (HighPrecMoney, Money)
+import Kernel.Types.Id
+
+data DriverFee = DriverFee
+  { id :: Id DriverFee,
+    shortId :: ShortId DriverFee,
+    driverId :: Id Driver,
+    govtCharges :: Money,
+    platformFee :: Money,
```

Comment:
It would have been better to create a single type for this, something like this
platformFee = PlatFormFee
where 
```
data PlatFormFee { fee :: Money, cgst :: HighPrecMoney, sgst :: HighPrecMoney }
```